data-esnext.js

```diff
@@ -1670,6 +1670,275 @@ exports.tests = [
     }
   ]
 },
+{
+  name: 'cancelable Promises',
+  category: 'proposal (stage 1)',
+  significance: 'large',
+  link: 'https://github.com/domenic/cancelable-promise',
+  subtests: [
+    {
+      name: 'Cancel, basic support',
+      exec: function(){/*
+        return (new Cancel('foo').message === 'foo');
+      */},
+      res: {
+      }
+    },
+    {
+      name: 'Cancel, does not require new',
```

Comment:
Nitpick: if a test name is a complete sentence (such as this, or "Map.prototype isn't an instance") then a comma isn't used in this way.